### DISCUSSION TITLE
How to use Hand Controlnet correctly

### DISCUSSION THREAD

**Discussion by dkqjhoifh** [Acly/krita-ai-diffusion#1267] [Q&A]
Krita's Hand ControlNet I don't know how to use it properly to fix my malformed hands。
![image](https://github.com/user-attachments/assets/5fed9bcb-99da-4f89-ae38-930dac493b5e)

**Comment by syl1130** [comment]
不能告诉你

**Comment by Marksusu** [comment]
你们可以了吧，唱什么大戏呢，好好画画不好吗？

![ComfyUI_temp_pyhpx_00001_](https://github.com/user-attachments/assets/03dd2810-d11f-4524-a443-3395f6b98d04)
